interfaces/functions.yaml

```diff
@@ -1011,6 +1073,13 @@ igraph_bipartite_game_gnm:
         INTEGER n1, INTEGER n2, INTEGER m, BOOLEAN directed,
         NEIMODE mode
 
+igraph_bipartite_game:
+    PARAMS: |-
+        OUT GRAPH graph, OPTIONAL OUT VECTOR_BOOL types,
+        ERDOS_RENYI_TYPE type, INTEGER n1, INTEGER n2, REAL p,
```

Comment:
Let's add defaults for `p` and `m` here; `p` is ignored for G(n,m) graphs and `m` is ignored for G(n,p) graphs so it does not make sense to ask the user to specify both.

Similarly, we should provde defaults for `directed` and `mode`